ports/ring-span-lite/portfile.cmake

```
@@ -16,14 +16,15 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 
 vcpkg_fixup_cmake_targets(
-    CONFIG_PATH lib/cmake/${PORT}
+    CONFIG_PATH "lib/cmake/${PORT}"

```

```suggestion
    CONFIG_PATH lib/cmake/${PORT}
```
It's unnecessary to add quotes for a relative path.